### DISCUSSION TITLE
Nera model support

### DISCUSSION THREAD

**Discussion by sircoynie** [Thomas55555/husqvarna_automower#530] [General]
I have successfully installed this integration for my new 430x Nera robot. Unfortunately it doesn’t appear to be fully supported yet as I do not have any long/lat entities for showing on the map. Anyone looking into this?

**Comment by Din-BH** [comment]
I had issues getting things to show up on the map when all coordinates painted were off the map image. If you turn on debug logging (see documentation to also turn it on for aioautomower) then you should see coordinates coming up in the detailed log. Do you?
